taskvine/src/manager/vine_manager.c

```diff
@@ -554,11 +554,13 @@ static vine_result_code_t get_completion_result(struct vine_manager *q, struct v
 			original->library_failed_count++;
 			original->time_when_last_failure = timestamp_get();
 		}
+		itable_remove(q->running_library_instances, t->task_id);
 		printf("Library %s failed on worker %s (%s)", t->provides_library, w->hostname, w->addrport);
```

Comment:
Use `debug(D_VINE|D_NOTICE)` to have consistent handling of messages to the user.  (It will also go to the log file)